### DISCUSSION TITLE
phone issue

### DISCUSSION THREAD

**Discussion by Rickshanelorie** [WalletConnect/walletconnect-monorepo#3294] [Developer Support]
everything working fine on pc but why on phone when I click connect it take me to the wallet but after I connect it don't redirect why and fix?

**Comment by glitch-txs** [comment]
Hi Rick, with which phone and wallet?

**Reply by Rickshanelorie** [reply]
Samsung and metamask it connect now but for some reason when I click any function it don't take me to metamask to confirm it is there's something in meta mask I need to do ??

**Reply by Rickshanelorie** [reply]
oh now metamask connect but i can't interact with the dapp on mobile unlike the pc and trust wallet won't connect for some reason so how can i fix this and thanks for you're time

**Comment by Rickshanelorie** [comment]
> Hi Rick, with which phone and wallet?
